### PR TITLE
chore(deps): bump solidity version to 0.8.22

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/blobstream-contracts/issues/245

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: src/Blobstream.sol [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.21;
+pragma solidity ^0.8.22;
```
[the remark this answers is not in the document]
Comment (evan-forbes) [reply src/Blobstream.sol:2]: > too recent to be trusted

lol should we wait to upgrade or do we need something from this version?

**Review by evan-forbes — APPROVED** [review]

File: src/Blobstream.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (SweeXordious) [reply src/Blobstream.sol:2]: it provides the unchecked arithmetics wrapper automatically and does all the checks itself... so it's more optimized

[the remark this answers is not in the document]
Comment (adlerjohn) [reply src/Blobstream.sol:2]: can you disable solidity version check for slither then?

**Review by adlerjohn — APPROVED** [review]

File: src/Blobstream.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (SweeXordious) [reply src/Blobstream.sol:2]: https://github.com/celestiaorg/blobstream-contracts/issues/248

**Pull request merged**